windows_10.json

```diff
@@ -65,7 +65,9 @@
         "./scripts/chocolatey.bat",
         "./scripts/chocolatey-packages.bat",
         "./scripts/firewall.bat",
-        "./scripts/doit.bat"
+        "./scripts/doit.bat",
+        "./scripts/clean.bat",
+        "./upstream/scripts/compact.bat"
```

Comment:
I've remember one issue that sometimes raises in the case of Windows guests VMs. At the very end of the build process virtualbox is unable to shutdown properly the VM. Something similar to this issue https://github.com/mitchellh/packer/issues/2889 . But it fixes, or never happend again, if you add the 'shutdown_timeout' option in the definition file.

@gtirloni  @avtar is that issue that either of you found in the builds of your boxes?